cli/src/torii/routing.rs

```diff
@@ -108,7 +109,9 @@ pub(crate) async fn handle_instructions(
     )
     .map_err(Error::AcceptTransaction)?;
     #[allow(clippy::map_err_ignore)]
-    let push_result = queue.push(transaction).map_err(|(_, err)| err);
+    let push_result = queue
+        .push(transaction, &sumeragi.wsv_mutex_access())
```

Comment:
at the moment the only way to get access to `wsv` outside of sumeragi is through a mutex?

Reply:
Yes. It is only used by torii for queries on the node from the outside world. The wsv that is access through the mutex is however not the one that sumeragi operates on. It is the 'public facing' wsv. So queries can take place while the wsv is being updated internally to sumeragi.